dbc_extract3/dbc/generator.py

```diff
@@ -1310,6 +1310,8 @@ class SpellDataGenerator(DataGenerator):
          356305,
          # Shredded Soul (Ebonsoul Vise)
          357785,
+		 # Dueling Form (Master Duelist's Chit)
+		 336236,
```

Comment:
```suggestion
         357785,
         # Dueling Form (Master Duelist's Chit)
         336236,
```
The simc code style uses spaces for indentation instead of tabs.